### PR TITLE
Rename references of "feature" to "loi" in firebase schema

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
Towards #1122 

@gino-m  PTAL?

### CODE REVIEW

**Review by gino-m — CHANGES_REQUESTED** [review]
I would consider calling these "lois" in Firestore instead of "locationOfInterest". Naming conventions I've seen in docs seem to prefer shorter, one word names. It should also be clear from code that these are associated with the LocationOfInterest class, hopefully making this name self-documenting.

@scolsen Wdyt? Does that sound reasonable?

**Comment by scolsen** [comment]
Sounds good to me!

I really slacked on my in-progress renaming and have some merging to sort out now 😅

**Comment by shobhitagarwal1612** [comment]
@gino-m PTAL! I've updated the references

File: gnd/src/main/java/com/google/android/gnd/persistence/remote/RemoteDataStore.java [hunk 1]
```diff
@@ -57,15 +57,15 @@ public interface RemoteDataStore {
   Maybe<TermsOfService> loadTermsOfService();
 
   /**
-   * Returns all features in the specified survey, then continues to emit any remote updates to the
-   * set of features in the survey until all subscribers have been disposed.
+   * Returns all lois in the specified survey, then continues to emit any remote updates to the set
```
Comment (gino-m) [inline gnd/src/main/java/com/google/android/gnd/persistence/remote/RemoteDataStore.java:60]: This is an acronym, so in comments this should be capitalized as "LOI" or "LOIs".

**Review by gino-m — APPROVED** [review]
LGTM, just one nit.

File: gnd/src/main/java/com/google/android/gnd/persistence/remote/RemoteDataStore.java (hunk 1 above)
Comment (shobhitagarwal1612) [reply gnd/src/main/java/com/google/android/gnd/persistence/remote/RemoteDataStore.java]: Done

**Pull request merged**